Feature/#508 change pipeline config documentation entities section (#540)
Avaiga/taipy-doc: docs/manuals/core/entities/sequence-mgt.md

The [Entities' creation](scenario-creation.md) section provides documentation on `Sequence^` creation. Now
that we know how to create a new `Sequence^`, this section focuses on describing the sequence's attributes
and utility methods.

> In this section, it is assumed that <a href="./code_example/my_config.py" download>`my_config.py`</a>

Reviewer: ```suggestion
In this section, it is assumed that the <a href="./code_example/my_config.py" download>`my_config.py`</a>
```

> module contains a Taipy configuration already implemented.

Reviewer: ```suggestion
module contains a Taipy configuration already implemented.
```

Reviewer 2: Or even:

"In this section, it is assumed that the \<a href="./code_example/my_config.py" download\>\`my_config.py\`\</a\>
module contains an already implemented configuration "

Reviewer: Yes you are right, let me fix it, thanks.

# Sequence attributes

A sequence is identified by a unique identifier `id` that Taipy generates. A sequence also holds
various properties accessible as an attribute of the sequence:

add data_nodes, parent_ids

- _**subscribers**_: The list of Tuples (callback, params) representing the subscribers.
- _**properties**_: The complete dictionary of the sequence properties. It includes a copy of the properties of

>   the sequence configuration, in addition to the properties provided at the creation and at runtime.

Reviewer: ```suggestion
  the sequence configuration, in addition to the properties provided at the creation and runtime.
```

> - _**tasks**_: The dictionary holds the sequence's various tasks. The key corresponds to the config_id of the

Reviewer 3: ```suggestion
- _**tasks**_: The dictionary holds the sequence's various `Task^`s. The key corresponds to the config_id of the
```

> - _**tasks**_: The dictionary holding the various tasks of the sequence. The key corresponds to the config_id of the
>   task while the value is the task itself.

Reviewer: ```suggestion
- _**tasks**_: The dictionary holds the sequence's various tasks. The key corresponds to the config_id of the
  task while the value is the task itself.
```

> - _**data_nodes**_: The dictionary holding the various data nodes of the sequence. The key corresponds to the

Reviewer 3: ```suggestion
- _**data_nodes**_: The dictionary holding the various `DataNode^`s of the sequence. The key corresponds to the
```

    data node's _config_id_ (while the value is the data node itself).
- _**owner_id**_: The identifier of the owner, which can be a scenario, cycle, or None.
- _**version**_: The string indicates the application version of the sequence to instantiate. If not provided,

>   the current version is used. Refer to the [version management](../versioning/index.md) page for more details.

Reviewer: ```suggestion
  the current version is used. For more details, refer to [version management](../versioning/index.md).
```

    import my_config

    scenario = tp.create_scenario(my_config.monthly_scenario_cfg, name="Monthly scenario")
    sales_sequence = scenario.sales_sequence
    sales_sequence.name = "Sequence for sales prediction"

    # There was no subscription, so subscribers is an empty list
    sequence.subscribers # []
    # The properties dictionary equals {"name": "Sequence for sales prediction"}. It
    # contains all the properties, including the `name` provided at the creation
    sequence.properties # {"name": "Sequence for sales prediction"}
    # The `name` property is also exposed directly as an attribute. It
    # equals "Sequence for sales prediction"
    sequence.name
    # The training task entity is exposed as an attribute of the sequence
    training_task = sequence.training
    # The predicting task entity as well
    predicting_task = sequence.predicting
    # The data nodes are also exposed as attributes of the sequence.
    current_month_data_node = sequence.current_month
    ```

# Get a sequence

There are two ways to retrieve a sequence:

The first method is by getting it from its parent scenario, using the sequence name as an attribute.

> The second method to access a sequence is from its id by using the `taipy.get()^` method.

Reviewer: ```suggestion
The second method to access a sequence is from its id using the `taipy.get()^` method.
```

```python linenums="1"
import taipy as tp
import my_config

scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
sequence = scenario.sales_sequence

sequence_retrieved = tp.get(sequence.id)
sequence == sequence_retrieved
```

Here the two variables `sequence` and `sequence_retrieved` are equal.

# Get all sequences

All the sequences can be retrieved using the method `taipy.get_sequences()^`. This method returns the list of all
existing sequences.

# Delete a sequence

> A sequence can be deleted by using `taipy.delete()^` which takes the sequence id as a parameter. The deletion is

Reviewer: ```suggestion
A sequence can be deleted using `taipy.delete()^` which takes the sequence id as a parameter. The deletion is
```

Reviewer 2: I would remove the "which takes the sequence id as a parameter" fragment. Useless. Like everywhere in the User Manual.
